packages/server/lib/browsers/cri-client.ts

```diff
@@ -11,12 +11,15 @@ import type { SendDebuggerCommand, OnFn, CdpCommand, CdpEvent } from './cdp_auto
 import type { ProtocolManagerShape } from '@packages/types'
 
 const debug = debugModule('cypress:server:browsers:cri-client')
+const debugVerbose = debugModule('cypress-verbose:server:browsers:cri-client')
 // debug using cypress-verbose:server:browsers:cri-client:send:*
-const debugVerboseSend = debugModule('cypress-verbose:server:browsers:cri-client:send:[-->]')
+const debugVerboseSend = debugModule(`${debugVerbose.namespace}:send:[-->]`)
 // debug using cypress-verbose:server:browsers:cri-client:recv:*
-const debugVerboseReceive = debugModule('cypress-verbose:server:browsers:cri-client:recv:[<--]')
+const debugVerboseReceive = debugModule(`${debugVerbose.namespace}:recv:[<--]`)
+// debug using cypress-verbose:server:browsers:cri-client:err:*
+const debugVerboseLifecycle = debugModule(`${debugVerbose.namespace}:ws`)
 
-const WEBSOCKET_NOT_OPEN_RE = /^WebSocket is (?:not open|already in CLOSING or CLOSED state)/
+const WEBSOCKET_NOT_OPEN_RE = /^WebSocket (?:connection closed|is (?:not open|already in CLOSING or CLOSED state))/
```

Comment:
Can we document what the various messages are that we're covering here? I always like to have examples around regexes for my future self.

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/29347/commits/a613bca09ada44aaf708e6e52cd8507e04272678